assets/app/styles/_log.less

```diff
@@ -0,0 +1,114 @@
+.log-title {
+  margin-top: 0;
+}
+.log-timestamp {
+  margin-bottom: 10px;
+}
+.log-view {
+  background-color: @log-bg-color;
+  min-height: 60px;
+  padding: 0;
+  position: relative;
+  pre, pre code {
+    background-color: transparent;
+    border: 0;
+    color: #98C1C8;
+    margin-bottom: 0;
+    overflow: visible;
+    padding: 0 10px;
+  }
+  .log-scroll {
+    text-align:right;
+    a {
+      padding-right:10px;
+    }
+    a:hover {
+      color: @link-hover-color-bright;
+    }
+  }
+  .log-scroll-top.affix {
+    position: fixed;
+    right: 30px;
+    top: 0;
+  }
+  .log-scroll-top.affix-top {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  .log-scroll-top.affix-bottom {
+
+  }
+  .log-scroll-bottom {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+  }
+  .log-view-output {
+    font-size: 12px;
+    padding: 20px 0 20px;
+    &:before {
+      content: '';
+      display: block;
+      height: 100%;
+      width: 1px;
+      background-color: lighten(@log-bg-color, 5%);
+      top: 0;
+      left: 60px;
+      position: absolute;
+    }
+    .spinner-inverse {
+      /*left: 50%;
+      position: absolute;
+      top: 15px;*/
+    }
```

Comment:
Its now block level with a `ng-show`, that way it stays at the bottom as new log lines appear.